### DISCUSSION TITLE
How to go back `n` number of screens in v6?

### DISCUSSION THREAD

**Discussion by curiosbasant** [react-navigation/react-navigation#10703] [Q&A]
I want to go back `n` number of screen from the current navigation history. How can I achieve that?

**Comment by raajnadar** [comment]
Use `navigation.navigate('Screen')` name in the history to move back in the history

**Reply by curiosbasant** [reply]
> Use `navigation.navigate('Screen')` name in the history to move back in the history

But idk the screen name. I just want to go back `n` screens. Would it be possible?

**Reply by raajnadar** [reply]
What is the use case? You can still send params to the next screen to know the previous screen or the screen in the history and navigate accordingly

**Comment by satya164** [comment]
https://reactnavigation.org/docs/stack-navigator#pop
